docs/releases/upgrading/upgrade-v10.0.md

```diff
@@ -65,8 +65,27 @@ pipenv run invenio communities rebuild-index
 
 This will ensure that all indices and their contents are based on the latest definitions and not out of date.
 
-As soon as the indices have been rebuilt, the entire migration is complete! :partying_face:
+#### Strict mappings
 
+Previously record and community mappins were dynamic, which means that any unknwon value could be added to it. This would be the case for custom elasticsearch dumper extensions. This dumpers need to be modified to dump those fields in new _custom fields_. Then the records and communities need to be re-indexed.
+
+\`\`\`shell
```

Comment:
```suggestion
```shell
```

Reply:
I removed the \'\'\' in a previous commit, if you still see it, it is a github bug. See: https://github.com/inveniosoftware/docs-invenio-rdm/pull/414/commits/54de2ce79d3839528317d578bdeaed702c2d4b87.